content/realtime/history.textile

```diff
@@ -134,14 +134,84 @@ Both the "@Channel@":/realtime/channels and "@Presence@":/realtime/presence obje
 
 h3(#persisted-history). Enabling persistent history
 
-By default, persisted history on channels is disabled and messages are only stored by the Ably service for two minutes in memory. If persisted history is enabled for the channel, then messages will "typically be stored for 24 - 72 hours on disk":https://support.ably.io/solution/articles/3000030059-how-long-are-messages-stored-for.
+By default, persisted history on channels is disabled and messages are only stored by the Ably service for two minutes in memory. If persisted history is enabled for the channel, then messages will "typically be stored for 24 - 72 hours on disk":https://support.ably.io/solution/articles/3000030059.
 
 Every message that is persisted to or retrieved from disk counts as an extra message towards your monthly quote. For example, for a channel that has persistence enabled, if a message is published, two messages will be deducted from your monthly quota. If the message is later retrieved from history, another message will be deducted from your monthly quota.
 
-To enable history on a channel, it is necessary to add a channel rule in the settings of your "application dashboard":https://support.ably.io/solution/articles/3000030053-how-do-i-access-my-app-dashboard. See the "documentation on channel rules":https://support.ably.io/solution/articles/3000030057-what-are-channel-rules-and-how-can-i-use-them-in-my-app for further information on what they are and how to configure them.
+To enable history on a channel, it is necessary to add a channel rule in the settings of your "application dashboard":https://support.ably.io/solution/articles/3000030053. See the "documentation on channel rules":https://support.ably.io/solution/articles/3000030057 for further information on what they are and how to configure them.
 
 h3(#continuous-history). Continuous history
 
+It is possible to obtain message history that is continuous with the realtime messages received on an attached channel, through the use of either "rewind":/realtime/messages, or the use of @untilAttach@ with History.
```

Comment:
This is kind of awkward to read, I think because of passive voice. Suggest rephrasing:

By using "rewind":/realtime/messages or History's @untilAttach@, it is possible to obtain message history that is continuous with the realtime messages received on an attached channel.

Also, I could be misunderstanding, but is continuous the right word to use here?